### PR TITLE
Drop support py<3.5 django<2.2

### PR DESCRIPTION
**Pull request by Mogost**
Related to #357 #358

### CODE REVIEW

File: tests/test_admin.py [hunk 1]
```diff
@@ -138,37 +137,38 @@ def test_inconsistent_fieldset_submit(self):
...
-        request = self.rf.get('/admin/constance/config/')
-        request.user = self.superuser
-        response = self.options.changelist_view(request, {})
-        response.render()
-        content_str = response.content.decode('utf-8')
-        self.assertGreater(
-            content_str.find('LONG_VALUE'),
-            content_str.find('INT_VALUE')
-        )
+    # TODO: Fix
```
Comment (Mogost) [inline tests/test_admin.py]: This test bothers me a bit.

Comment (camilonova) [reply tests/test_admin.py]: Can you please work on this so I can merge it?

Comment (Mogost) [reply tests/test_admin.py]: Ok. Done

**Comment by camilonova** [comment]
Thank you

**Comment by Natureshadow** [comment]
Can someone please upload a new version to PyPI with full Django 3.0 support?

**Pull request merged**